sqlchain/tx.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 The ThunderDB Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sqlchain
+
+import (
+	"github.com/thunderdb/ThunderDB/crypto/hash"
+	"github.com/thunderdb/ThunderDB/crypto/signature"
+)
+
+type TX struct {
+	version int32
+	TxIn    []*TxIn
+	TxOut   []*TxOut
+}
+
+type TxIn struct {
+	previousHash hash.Hash
+	Signee       *signature.PublicKey
+	Signature    *signature.Signature
+}
+
+type TxOut struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type TxOut should have comment or be unexported